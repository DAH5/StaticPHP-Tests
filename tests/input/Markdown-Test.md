The quick brown fox jumped over the lazy dog.

The quick brown fox
jumped over the lazy dog.

# Heading 1
## Heading 2
### Heading 3
#### Heading 4
##### Heading 5
###### Heading 6

**bold** or __bold__
*italic* or _italic_
~~strikethrough~~

[Link Text](http://hostname.tld/resource)

[Link Text](http://hostname.tld/resource "Title Text")

![Alt Text](http://hostname.tld/image.png)

![Alt Text](http://hostname.tld/image.png "Title Text")

`code`

- Item 1
- Item 2
- Another List Item
- Yet Another List Item