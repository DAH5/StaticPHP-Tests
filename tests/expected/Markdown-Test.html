The quick brown fox jumped over the lazy dog.

The quick brown fox
jumped over the lazy dog.

<h1>Heading 1</h1>
<h2>Heading 2</h2>
<h3>Heading 3</h3>
<h4>Heading 4</h4>
<h5>Heading 5</h5>
<h6>Heading 6</h6>

<strong>bold</strong> or <strong>bold</strong>
<em>italic</em> or <em>italic</em>
<del>strikethrough</del>

<a href="http://hostname.tld/resource" target="_blank" rel="noopener noreferrer">Link Text</a>

<a href="http://hostname.tld/resource" title="Title Text" target="_blank" rel="noopener noreferrer">Link Text</a>

<img src="http://hostname.tld/image.png" alt="Alt Text">

<img src="http://hostname.tld/image.png" alt="Alt Text" title="Title Text">

<code>code</code>

<ul>
<li>Item 1</li>
<li>Item 2</li>
<li>Another List Item</li>
<li>Yet Another List Item</li>
</ul>